src/commands/integration-test-export-logs.yaml

```diff
@@ -0,0 +1,9 @@
+steps:
+  - run:
+      name: "architect/integration-test-export-logs: Export cluster logs"
+      command: |
+        kind export logs ./kind/logs
+      when: always
```

Comment:
I think it would be good to move `when: always` part to the job to make it more visible at the high level.